packages/lib/timeFormat.ts

```diff
@@ -6,8 +6,16 @@
  */
 import { localStorage } from "@calcom/lib/webstorage";
 
+const is24hLocalstorageKey = "timeOption.is24hClock";
+
+export const setIs24hClockInLocalStorage = (is24h: "true" | "false") =>
+  localStorage.setItem(is24hLocalstorageKey, is24h);
```

Comment:
Is there a reason that we can't receive true/false as a boolean and just save it as is24h.toString()?

Reply:
I think we should be safe as anything with cookies, localStorage, and query params is expected to be string value saved. We can aim for ease of use in this case. You code once and use it everywhere, instead of using === "true" when being used in other places.